app/Http/Controllers/PostController.php

```diff
@@ -228,4 +228,18 @@ private function generateDataUri($img_obj)
 
         return $data_uri;
     }
+
+    public function showCalendar(Request $request)
+    {
+    $date = $request->input('date', now()->format('Y-m-d'));
+
+    $posts = Post::whereHas('postCategories', function($query) {
+                     $query->where('category_id', 2);
```

Comment:
is the "2" intentional?

Reply:
Categories, 2 is the category "event" and I want to display only events.

Reply:
got it. thankyou